src/esbmc/esbmc_parseoptions.cpp

```diff
@@ -1592,6 +1593,12 @@ bool esbmc_parseoptionst::process_goto_program(
       value_set_analysis.update(goto_functions);
     }
 
+    // unwind loops
```

Comment:
There is already a class member `options` that holds all the options, no need to create another one.